BootloaderCorePkg/Include/Library/SmbiosInitLib.h

```diff
@@ -0,0 +1,66 @@
+/** @file
+
+  Copyright (c) 2017 - 2019, Intel Corporation. All rights reserved.<BR>
+  This program and the accompanying materials
+  are licensed and made available under the terms and conditions of the BSD License
+  which accompanies this distribution.  The full text of the license may be found at
+  http://opensource.org/licenses/bsd-license.php.
+
+  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+
+**/
+
+#ifndef __SMBIOS_INIT_LIB_H__
+#define __SMBIOS_INIT_LIB_H__
+
+#include <IndustryStandard/SmBios.h>
+
+
+typedef struct {
+  UINT16    Type;
```

Comment:
Why is it UINT16 instead of UINT8 ?

Reply:
Changed the field to UINT8, and pragma packed the struct